docs/markdown/intro/README.md

```diff
@@ -129,7 +129,7 @@ Then, register BootstrapVue in your app entry point:
 ```js
 // app.js
 import Vue from 'vue'
-import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'
+import BootstrapVue, { IconsPlugin } from 'bootstrap-vue'
```

Comment:
`BootstrapVue` is also available as a named export,

It might be better to keep the named import, and add a default import on the same line that is set as a plugin

Reply:
You might be using an older  version of BootstrapVue (i.e. earlier 2.0.0-rc.xx releases) which does not have the named export for BootstrapVue

Reply:
well, I just installed it... looking at the package.json of the bootstrap-vue in node_modules it says

``` "version": "2.2.0",```

Reply:
Then this is a bug with the `/src/index.d.ts` file (see https://github.com/bootstrap-vue/bootstrap-vue/pull/4588#issuecomment-572805406)

Reply:
should I kill this PR?

Reply:
Or alter it to fix just the `src/index.d.ts` file only (and restore the README.md file)

Reply:
Or if you want, I can create a new PR that fixes the index.d.ts file